iotqatools/simulators/listen/package.json

```diff
@@ -22,8 +22,8 @@
     },
     "dependencies": {
         "moment": "*",
-        "dateformat": "*",
-        "mqtt": "*"
+        "dateformat": "4.5.1",
+        "mqtt": "4.2.8"
```

Comment:
Maybe we should do the same with moment library... a similar problem may arise in the future when we didn't remember what happened today with this :)

Reply:
fixed in https://github.com/telefonicaid/iotqatools/pull/184/commits/1b5a85560e232f0132d0ac2e038d353007f5f534